app/adapters/job.js

```diff
@@ -1,7 +1,7 @@
 import V3Adapter from 'travis/adapters/v3';
 
 export default V3Adapter.extend({
-  includes: 'build.request,build.commit,build.created_by',
+  includes: 'build.request,build.commit,build.created_by,job.config',
```

Comment:
I think that this line should be reverted. The whole idea of fetching config lazily is to not always include it as it is adding an extra work for the server and we only need it when viewing a job.